gradle/shared-with-buildSrc/mirrors.settings.gradle.kts

```diff
@@ -34,7 +35,12 @@ val mirrorUrls: Map<String, String> =
         name to url
     } ?: emptyMap()
 
-fun isEc2Agent() = java.net.InetAddress.getLocalHost().getHostName().startsWith("ip-")
+
+fun isEc2Agent() = java.net.InetAddress.getLocalHost().hostName.startsWith("ip-")
```

Comment:
Can you use the `HostnameLookup` service instead?

Reply:
Two questions: 

- What's the advantage of `HostnameLookup`?
- Is it more convenient to be used in init script?

Reply:
> * What's the advantage of `HostnameLookup`?

It is faster.

> * Is it more convenient to be used in init script?

How easy is it to obtain a service in an init script?